packages/geospatial/src/components/CoreDataPlace.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
I think if you add the `// @flow` annotation to the top of this file, it will like the syntax a lot more. There's also a few ESLint errors on the formatting, but less concerned about those.

Reply:
Done, but even with Flow enabled, I'm still getting red squiggles under every TypeScript statement, and Flow shows all the TS stuff as "uncovered area". I'll enquire with Camden.